pyrokinetics/normalisation.py

```diff
@@ -672,13 +672,15 @@ def set_lref(
 
         if local_geometry:
             minor_radius = local_geometry.a_minor
-            major_radius = local_geometry.Rmaj
-
-        # FIXME: We could also set this from minor_radius * aspect_ratio
-        major_radius = major_radius or 0.0
+            aspect_ratio = local_geometry.Rmaj
```

Comment:
I'm really not sure about this -- surely this is only true if `a_minor == 1.0`? In which case, it would still be correct to have `aspect_ratio = major_radius / minor_radius`

Reply:
At the moment `Rmaj` is defined as `major_radius/minor_radius`, so it's the normalised major radius.

Eventually we should have the normalising length scale be an option but we haven't set up units in the `LocalGeometry` and that's probably a separate PR. But currently this is always the aspect ratio.

https://github.com/pyro-kinetics/pyrokinetics/blob/fb278e4ab7ee28a9899beca0757013368086fb4e/pyrokinetics/local_geometry/LocalGeometry.py#L159